cmd/vclusterctl/cmd/delete.go

```diff
@@ -182,45 +200,66 @@ func (cmd *DeleteCmd) Run(cobraCmd *cobra.Command, args []string) error {
 }
 
 func (cmd *DeleteCmd) prepare(vClusterName string) error {
-	vCluster, err := find.GetVCluster(cmd.Context, vClusterName, cmd.Namespace)
-	if err != nil {
-		return err
-	}
+	if cmd.ignoreNotFound {
```

Comment:
The `prepare` method is also used to create kubeconfig, and it does that using Vclusters. But if the Vcluster isn't present, it will panic and error out (with nil pointer). To avoid that and to still be able to create kubeclient, I am checking if the flag `--ignore-not-found` is present and then creating it without using vcluster.